prompt_toolkit/input/ansi_escape_sequences.py

```diff
@@ -267,6 +267,8 @@
     "\x1b[1;9B": (Keys.Escape, Keys.Down),
     "\x1b[1;9C": (Keys.Escape, Keys.Right),
     "\x1b[1;9D": (Keys.Escape, Keys.Left),
+    "\x1b[1;10C": Keys.EscapeShiftRight,
```

Comment:
Hi @dannysepler,

These changes look fine for me, except for one thing. "escape" is always considered a separate key in prompt_toolkit, unlike control or shift. The reason is that people can type "escape" first, and later the arrow key, or they can do "alt" + arrow key. For the parser, the escape is a separate key.

So, this means that like the above key bindings, this should be a tuple (an ANSI sequence that corresponds with multiple keys):
```python
    "\x1b[1;10C": (Keys.Escape, Keys.ShiftRight)
```